app/scripts/metamask-controller.js

```diff
@@ -5750,10 +5750,9 @@ export default class MetamaskController extends EventEmitter {
           this.permissionController.requestPermissions(
             { origin },
             {
-              ...(process.env.CHAIN_PERMISSIONS &&
-                requestedPermissions[RestrictedMethods.eth_accounts] && {
-                  [PermissionNames.permittedChains]: {},
-                }),
+              ...(requestedPermissions[RestrictedMethods.eth_accounts] && {
+                [PermissionNames.permittedChains]: {},
+              }),
```

Comment:
> do we anticipate a user trying to request permittedChains by itself? if so we probably need to populate an empty eth_accounts object here too? @adonesky1

🤔 I think whether or not we encourage it we should include an accounts permission as part of the request to avoid confusion/frustration